### PR TITLE
[bug] 7236: Secondary contact accessing case information

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/integration-tests/externalUserViewsLegacySealedDocuments.test.js [hunk 1]
```diff
@@ -83,7 +90,7 @@ describe('External user views legacy sealed documents', () => {
   });
 
   loginAs(test, 'petitioner2@example.com');
-  unassociatedUserViewsCaseDetailForCaseWithLegacySealedDocument(test);
```
Comment (rachaelparris) [inline web-client/integration-tests/externalUserViewsLegacySealedDocuments.test.js:86]: I think we do need to continue testing an unassociated petitioner viewing a legacy sealed document. Is there another way you could set this up so we can test all the different scenarios? We might need to add a 3rd petitioner login for local.

File: web-client/src/presenter/actions/getCaseAssociationAction.test.js [hunk 2]
```diff
@@ -3,21 +3,17 @@ import { applicationContextForClient as applicationContext } from '../../../../s
 import { getCaseAssociationAction } from './getCaseAssociationAction';
 import { presenter } from '../presenter-mock';
 import { runAction } from 'cerebral/test';
-
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/getCaseAssociationAction.test.js:6]: What happened to all these lines?

Comment (sutt0n) [reply web-client/src/presenter/actions/getCaseAssociationAction.test.js:6]: eslint would be my guess

File: web-api/storage/scripts/createUsers.js [hunk 3]
```diff
@@ -50,6 +53,20 @@ module.exports.createUsers = async () => {
           usersByEmail[userCreated.email] = userCreated;
         });
       }
+
+      if (userRecord.role === ROLES.petitioner) {
```
Comment (rachaelparris) [inline web-api/storage/scripts/createUsers.js:57]: Is this change relevant to this bug?

File: web-client/src/presenter/actions/getCaseAssociationAction.test.js (hunk 2 above)
Comment (rachaelparris) [reply web-client/src/presenter/actions/getCaseAssociationAction.test.js:6]: 😱

File: web-api/storage/scripts/createUsers.js (hunk 3 above)
Comment (sutt0n) [reply web-api/storage/scripts/createUsers.js:57]: It's relevant for ensuring our local petitioners from `users.json` are indexed with the `user-email|` primary key and migrations -- e.g. if a case's `contactPrimary` is `petitioner@example.com` and `contactSecondary` is `petitioner2@example.com`

**Review by kkoskelin — APPROVED** [review]
